resources/log4j.properties

```diff
@@ -20,5 +20,6 @@ log4j.appender.metabase=metabase.logger.Appender
 log4j.logger.metabase=INFO
 log4j.logger.metabase.db=DEBUG
 log4j.logger.metabase.driver=DEBUG
+log4j.logger.metabase.middleware=DEBUG
```

Comment:
Logging API calls is useful because it let's us see what status codes get returned (e.g. 200 vs 500) which helps us debug issues other people are seeing. It also keeps us conscious of how long API calls take (which calls are fast vs which are slow). Plus, it's really not useful to have DB calls logged (which we're doing) but not the API endpoints that trigger them.